Artiom/src/main/java/Circle_Area.java

```diff
@@ -0,0 +1,8 @@
+public class Circle_Area {
+    public static void main(String[] args) {
```

Comment:
It's a good practice to move logic to the separate method and use main method for high level operations.
So here you can create a separate method which will accept one parameter (radius) and return the area.
Use this method from main to print calculated area.

Reply:
done